data/delegates.json

```diff
@@ -258,5 +258,22 @@
     "poolMember": false,
     "required": false,
     "featured": false
+  },
+  {
+    "delegateName": "elevate",
+    "delegateAddress": "17890508407355636952L",
+    "proposal": "",
```

Comment:
Hey, thanks for your submission. A proposal is not a must-have, but in general, it's the staging point for everyone to learn about you and yourself and your projects. Also, a forum proposal can be automatically linked to your delegate on the block explorer.

Reply:
hey @4fryn :)

I already added the proposal! And thanks for the tip with the block explorer!